### PR TITLE
Wednesday (module 2) - Update & Delete

### PR DESCRIPTION
**Pull request by cameronjacoby**
ALL: This is still a work-in-progress, but the bulk of the content is here. Wanted to go ahead and pull-request it so everyone can see what @bgveenstra and I are planning on covering tomorrow afternoon.

They will have a CRUD API started from the morning lecture with CREATE & READ. We're going to walk through UPDATE in class, then have them implement DELETE on their own for the challenges / HW.

### CODE REVIEW

**Comment by bgveenstra** [comment]
no longer WIP! README should match solution readme

File: w3_d3_2_update_and_delete/README.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+```
+
+## Challenges (& Tonight's Homework)
+
+Implement the "D" (delete) in CRUD in the RESTful API you started this morning.
+
+### Requirements
+
+1. A user should be able to click a button on the page to indicate they'd like to delete an item.
+2. Your client-side JavaScript should register clicks on your delete button and send an DELETE request (via AJAX) to the server.
```
Comment (janephilipps) [inline w3_d3_2_update_and_delete/README.md]: should be "a" DELETE request right? I would fix this myself, but can't remember how to do it! Next time :)

**Comment by janephilipps** [comment]
This looks AMAZING! You ladies rock :+1:

**Comment by cameronjacoby** [comment]
@janephilipps Thanks so much for reviewing!

**Comment by cameronjacoby** [comment]
FYI for everyone -- Brianna and I moved the client-side requirements to the stretch challenges and are only going to focus on server-side update/delete during the modules.

This doesn't change the notes at all -- the students can reference the client-side examples in the notes if they are working on the stretch challenges.

**Pull request merged**